src/plugins/cosmos/components/modals/Staking/views/StakeBroadcast.tsx

```diff
@@ -54,9 +54,7 @@ export const StakeBroadcast = ({
 
   const { handleStakingAction } = useStakingAction()
   const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
-  const validatorInfo = useAppSelector(state =>
-    selectSingleValidator(state, accountSpecifier, validatorAddress),
-  )
+  const validatorInfo = useAppSelector(state => selectSingleValidator(state, { validatorAddress }))
```

Comment:
I have it removed from props in the [final PR](https://github.com/shapeshift/web/pull/1652) but you're right, since we're removing the need for `accountSpecifier` in this PR, might as well remove it here!